src/MicrosoftTeams.ts

```diff
@@ -588,6 +590,49 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    ensureInitialized();
+    printCapabilityEnabled = true;
+    /**
+     * adding ctrl+P and cmd+P handler
+     */
+    document.addEventListener("keydown", function(event: KeyboardEvent): void {
+      if ((event.ctrlKey || event.metaKey) && event.keyCode === 80) {
+        microsoftTeams.print();
+        event.cancelBubble = true;
+        event.preventDefault();
+        event.stopImmediatePropagation();
+      }
+    });
+  }
+
+  /**
+   * Registers a handler for print.
+   * default print handler
+   */
+  export function print(): void {
+    if (printCapabilityEnabled) {
```

Comment:
Is printCapabilityEnabled meant to gate the entire printing feature, or just so we don't register ctrl-P/cmd-P unnecessarily? Not sure why we need to check for printCapabilityEnabled in print(), when the page can easily bypass it by calling window.print() (or the custom handler) directly.

Reply:
I was thinking to make it backward compatible so that to get default Print support app need to call enablePrintCapability. I am adding check in print function to make sure app developer call enablePrintCapability before using it either explicit call or Ctr+P